charts/scalardb/templates/scalardb/deployment.yaml

```diff
@@ -80,13 +80,28 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.scalardb.resources | nindent 12 }}
+          startupProbe:
+            exec:
+              command:
+              - /usr/local/bin/grpc_health_probe
+              - -addr=:60051
+            failureThreshold: 60
+            periodSeconds: 5
```

Comment:
I set the timeout value to 300sec (`failureThreshold * periodSeconds = 300sec`). If the pod won't be `Ready` state after 300sec, Kubernetes terminates the pod.